### PR TITLE
[GR-1205] Add Home button to navbar on reports

### PR DESCRIPTION
**Pull request by avarsava**
Please test to make sure this looks OK on your display! I only have my laptop to test on. It looks good at a variety of window sizes for me.

### CODE REVIEW

**Review by slazicoicr — APPROVED** [review]
The home button seems to disappear when i make the window narrow (~1/4 of my screen). Don't think this is an issue as the graphs themselves become too narrow at this width.

image

**Comment by avarsava** [comment]
> The home button seems to disappear when i make the window narrow (~1/4 of my screen). Don't think this is an issue as the graphs themselves become too narrow at this width.
> 

Yeah I think I'm OK saying 'there is a width at which Dashi becomes unusably narrow' and it seems to be 1/5th of my screen width

File: application/dash_application/known_pages_router.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def menu_item(label, link):
     return dbc.NavbarSimple(
         id="navbar",
         children=[
+            # Simple Navbar only lets you add content to the right side. This CSS forces the button out of the div 
+            html.A(html.Button("Home", style={"position": "fixed", "left": "0", "margin-left": "15px", "margin-top": "4px"}), href="/.."),
```
Comment (apmasell) [inline application/dash_application/known_pages_router.py]: I think either `"/"` or `".."`, but both is not doing anything.

**Review by apmasell — APPROVED** [review]

File: application/dash_application/known_pages_router.py (hunk 1 above)
Comment (avarsava) [reply application/dash_application/known_pages_router.py]: Well that's weird, because it works for me. What's your environment like? I'm on firefox 76

Comment (apmasell) [reply application/dash_application/known_pages_router.py]: Oh, it works, it's just _go to the parent of the root directory_ doesn't mean anything beyond _go to the root directory_.

Comment (avarsava) [reply application/dash_application/known_pages_router.py]: Oh, didn't know a link to / would resolve, figured the link would be constructed like 'localhost:5000/single-lane-ts?run=my_cool_run/..'

**Pull request merged**